libs/services/src/SlippageService/SlippageService.ts

```diff
@@ -5,6 +5,8 @@ import {
 } from '@cowprotocol/repositories';
 import { injectable, inject } from 'inversify';
 
+const DEFAULT_SLIPPAGE_BPS = 200;
```

Comment:
Won't this change impact CoW Swap as it's currently polling the slippage endpoint?

Reply:
Let me fix it by making another service which will be the one used by default until the other one is finished

Reply:
I'll do it in another PR

Reply:
https://github.com/cowprotocol/bff/issues/63 will fix soom, I'll try to consolidate things first